apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -781,6 +784,25 @@ Optional arguments:
 "
 `;
 
+exports[`CommandLineHelp prints the help for each action: update-build-cache-credentials 1`] = `
+"usage: rush update-build-cache-credentials [-h] [-i]
+                                           [--credential CREDENTIAL_STRING]
+                                           [-d]
```

Comment:
Can we make this name more concise? It is pretty long.  Some ideas:

- `rush authorize-build-cache`
- `rush update-credentials`
- `rush set-cloud-credentials`
- `rush authorize-cloud-provider`

If one day Rush has 3 different features that all use cloud credentials, it seems unlikely that they would each need their own dedicated Rush command.  It may even be the case that they all share the same credential.